src/libraries/System.Speech/ref/System.Speech.csproj

```diff
@@ -0,0 +1,8 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
```

Comment:
IIRC we need a net461 configuration as well to avoid binding redirect issues on Desktop. cc @joperezr

Reply:
OK disregard, I just saw that netstandard2.0 is a PNSE.